conf/ingress.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: orsp-pub-dev-ip
+  labels:
+    app: ${REPO_NAME}
+  name: ${REPO_NAME}-ingress
+  namespace: default
+spec:
+  backend:
+    serviceName: ${REPO_NAME}-service
+    servicePort: http
+  rules:
+  - host: ui.dsp-orsp-dev.broadinstitute.org
+    http:
+      paths:
+      - backend:
+          serviceName: ${REPO_NAME}-service
+          servicePort: http
+  tls:
+  - hosts:
+    - ui.dsp-orsp-dev.broadinstitute.org
```

Comment:
How permanent is this domain name? Who manages `dsp-orsp-dev` ?

Reply:
It is as permanent as we wish, I guess, dsp-orsp-dev DSN Zone (Cloud DNS) is under control of orsp-pub project, us, in a similar way as dsp-duos-dev is under broad-duos-dev project. That is part of the work done by Henry Ferrrara's team after my trip to Boston.